### PR TITLE
Create README.md for ShowLocationOnMap automation

### PR DESCRIPTION
**Pull request by richardbluestone**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by richardbluestone** [comment]
@altmannyarden Not sure if you need to approve here, but you can review if required.

File: Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md
Comment (ali-sawyer) [inline Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:3]: I'd suggest moving the details about adding a General Purpose Dynamic Section to the indicator layout from https://github.com/demisto/content-docs/pull/919 to this page, because these steps are specific to the `ShowLocationOnMap` automation.

File: Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md
Comment (ali-sawyer) [inline Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:11]: The required tag is `dynamic-indicator-section`.

Screen Shot 2022-01-08 at 10 06 02 PM
.

**Review by ali-sawyer — APPROVED** [review]
Looks good! Added a couple comments.

File: Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md
Comment (richardbluestone) [reply Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:3]: Initially I was going to place the content in the `ShowLocationOnMap` automation but there seems to be a convention not to include it. None of the automation docs have additional information. I'm reluctant to add it, so that is why I thought of an additional doc and we'll have an xref to this one.

**Review by richardbluestone — COMMENTED** [review]
Made the changes as requested

File: Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md
Comment (altmannyarden) [inline Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:1]: Displays a geographical location using Google Maps.

we are not displaying the location in the field - displays in the indicator layout (that using the `geolocation` field).

File: Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md
Comment (altmannyarden) [inline Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:3]: You need to setup Google Maps to use this automation in Cortex XSOAR.To view the map, you will need to edit the indicator layout, add a new "General Purpose Dynamic section" and choose the "ShowLocationOnMap" automation.
After those changes, you can see the location in the edited / indicator layout.

**Review by richardbluestone — COMMENTED** [review]
Made the changes requested by Yarden

File: Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md
Comment (altmannyarden) [inline Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:1]: Using the geolocation field... the location you will see in the layout (in the chosen dynamic section)

maybe we should add a reference to the article you are working on 
https://github.com/demisto/content-docs/pull/919#issuecomment-1008256723

Comment (richardbluestone) [reply Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:1]: Yes I agree. I thought we can't reference it yet until the article is pushed to the master?

Comment (altmannyarden) [reply Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:1]: True, :) So the question is - if should we merge it now and add the reference later? Or wait until we merge the article?

Comment (richardbluestone) [reply Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md:1]: I think we should probably wait until we merge the article. Also this article is not validating - is that because we don't have approvals yet?

**Comment by richardbluestone** [comment]
@altmannyarden The validation keeps failing. Is it because we don't have a banner that says the script is part of the Command Scripts pack, like this: 
image
I'm not sure how to add it. Can you have a look? Thanks

**Comment by altmannyarden** [comment]
@richardbluestone why do you think that? I see its failed because you need to update your pr from maser (need to update the circle.ci).

**Pull request closed without merging**
